README.rst

```diff
@@ -41,25 +41,33 @@ you access to the API client that you want:
 
     import signalfx
 
-    sfx = signalfx.SignalFx()
+    sfx = signalfx.SignalFx(api_endpoint='https://api.{REALM}.signalfx.com',
+            ingest_endpoint='https://ingest.{REALM}.signalfx.com',
+            stream_endpoint='https://stream.{REALM}.signalfx.com')
+
 
     # For the ingest API
-    ingest = sfx.ingest('ORG_ACCESS_TOKEN')
+    ingest = sfx.ingest('ORG_TOKEN')
 
     # For the REST API
-    rest = sfx.rest('USER_ACCESS_TOKEN')
+    rest = sfx.rest('API_TOKEN')
 
     # For the SignalFlow API
-    flow = sfx.signalflow('USER_ACCESS_TOKEN')
+    flow = sfx.signalflow('ACCESS_TOKEN')
 
-As you may have noticed, you need to specify an access token when requesting
+If no endpoints are set manually, this library uses the ``us0`` realm by default. 
+If you are not in this realm, you will need to explicitly set the
+endpoint urls above. To determine if you are in a different realm and need to
+explicitly set the endpoints, check your profile page in the SignalFx 
+web hpplication. You will also need to specify an access token when requesting
 one of those clients. For the ingest client, you need to specify your
 organization or team's API access token (which can be obtained from the
```

Comment:
There is no such thing as a team API access token. Some teams may be told to use a specific token, but that's not anything we enforce.  So this should just be "your organization's access token" (or whatever wording you used elsewhere)